packages/core/src/sso/types/index.ts

```diff
@@ -11,12 +11,3 @@ export abstract class SingleSignOn {
   abstract data: SsoConnector;
   abstract getConfig: () => Promise<JsonObject>;
 }
-
-export enum SsoProviderName {
-  OIDC = 'OIDC',
-  SAML = 'SAML',
-}
-
-export type SupportedSsoConnector = Omit<SsoConnector, 'providerName'> & {
-  providerName: SsoProviderName;
-};
```

Comment:
Better keep the definition of SsoProviderName within the SSO factories' directory.

Reply:
Updated.